app/client/src/entities/IDE/constants.ts

```diff
@@ -5,6 +5,16 @@ export enum EditorState {
   LIBRARIES = "LIBRARIES",
 }
 
+export enum SidebarTopButtonTitles {
```

Comment:
```suggestion
export const SidebarTopButtonTitles = {
```

Lets keep both top and bottom export types same, and I think const is a more readable choice.